utils/ansible/ftest/templates/daos-launch.sh.j2

```diff
@@ -0,0 +1,236 @@
+#!/bin/bash
+# shellcheck=disable
```

Comment:
(lint) Couldn't parse this shellcheck directive. Fix to allow more checks. [SC1073]
(lint) Fix any mentioned problems and try again. [SC1072]